public/img/logo.svg

```diff
@@ -0,0 +1,16 @@
+<svg version="1.1" preserveAspectRatio="xMinYMin meet" width="100%" height="100%" viewBox="0 0 655 260" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We should probably move the `width` and `height` attributes out of the `<svg/>` tag so that they don't cause conflicts with the containing element.